docs/workflow/requirements/freebsd-requirements.md

```diff
@@ -65,3 +64,46 @@ sudo pkg install --yes libunwind icu libinotify lttng-ust krb5 cmake openssl nin
 Additionally, working dotnet cli with SDK is needed. On other platforms this would be downloaded automatically during build but it is not currently available for FreeBSD.
 It needs to be built once on supported platform or obtained via community resources.
 
+Running on FreeBSD
+-------------------
+Install the following packages:
+- icu
+- libunwind
+- lttng-ust (optional, debug support)
+- krb5
+- openssl (optional, SSL support)
```

Comment:
Isn't openssl part of base OS?
```
furt@d13:~ $ which openssl
/usr/bin/openssl

```

Reply:
"yes but"
Yes! But that version is for the OS and things from ports should use other things from ports. 

Which can be confusing but generally...

If it ships with the base OS its for the base only (e.g., openssl, llvm/clang, libunwind) or for building the OS. If it comes from ports (something external from the base OS) , it should use programs from ports

Reply:
oh, well. I'm not going to argue.